package.json

```diff
@@ -26,33 +26,45 @@
     "test": "yarn run test:prepare && yarn run cy:run"
   },
   "devDependencies": {
-    "@playkit-js/common": "1.5.2-canary.0-f23d183",
-    "@playkit-js/kaltura-player-js": "3.17.16-canary.0-db12b53",
-    "@playkit-js/ui-managers": "1.4.3-canary.0-b33eab0",
+    "@babel/core": "^7.23.7",
+    "@babel/plugin-proposal-decorators": "^7.10.5",
+    "@babel/plugin-transform-react-jsx": "^7.23.4",
+    "@babel/plugin-transform-runtime": "^7.23.7",
+    "@babel/preset-env": "^7.23.8",
+    "@babel/preset-typescript": "^7.23.3",
+    "@babel/runtime": "^7.23.8",
+    "@playkit-js/browserslist-config": "1.0.8",
+    "@playkit-js/common": "1.5.19",
+    "@playkit-js/kaltura-player-js": "3.17.24",
+    "@playkit-js/ui-managers": "1.6.0",
     "@typescript-eslint/eslint-plugin": "^5.58.0",
     "@typescript-eslint/parser": "^5.58.0",
+    "babel-loader": "^9.1.3",
     "conventional-github-releaser": "3.1.3",
-    "css-loader": "^6.7.3",
-    "cypress": "^13.6.4",
+    "css-loader": "^6.9.0",
+    "cypress": "^13.7.3",
```

Comment:
better use explicit cypress 13.13.1 to avoid the firefox timeout issue in case the caret gets resolved to another version